docker-compose-v1.yml

```diff
@@ -6,9 +6,9 @@ db:
     - /etc/localtime:/etc/localtime:ro
   # uncomment the following to enable backup
   environment:
-    - MM_USERNAME=mmuser
-    - MM_PASSWORD=mmuser_password
-    - MM_DBNAME=mattermost
+    - POSTGRES_USER=mmuser
```

Comment:
Could you add a comment let people know those values has to be the same as environment variables in app service?